old/index.html

```diff
@@ -115,7 +115,7 @@
                     If you don't know what JPEG XL is, <a href="why-jxl.html">check out its key advantages!</a>
                 </p>
                 <p>
-                    Want to update this site? <a href="https://github.com/jxl-community/jxl-community.github.io">Make a pull request!</a>
+                    This is the old site, check out the new site <a href="https://github.com/jxl-community/jxl-community.github.io">here!</a>
```

Comment:
```suggestion
                    This is the old site, check out the <a href="../">new site !</a>.
```

Maybe better to point to the new site itself, not the github repo (there's a link to the github repo at the bottom of the new site).

Reply:
Sorry, forgot to change the link

Reply:
Lesson learned : don’t do web dev when you just woke up